README.md

```diff
@@ -1,8 +1,88 @@
 # BitmovinPlayerCore
 
-**This package is currently under active development and is not yet ready for production use.**
+- [Installation](#installation)
+    - [CocoaPods](#using-cocoapods)
+    - [Swift Package Manager](#using-swift-package-manager)
+    - [Add the Player SDK to the Project directly](#add-the-player-sdk-to-the-project-directly)
+- [Getting Started](#getting-started)
+    - [Configure the license key](#configure-the-license-key)
+- [Samples](#samples)
+- [Documentation and Resources](#documentation-and-resources)
+
+The `BitmovinPlayerCore` package is the standalone `Player` without additional dependencies to other modules such as `BitmovinAnalytics`.
+
+## Installation
+To add the `BitmovinPlayerCore` SDK as a dependency to your project, you have three options: Using CocoaPods, Swift Package Manager or adding the SDK bundle directly.
+
+### Using [CocoaPods](https://cocoapods.org/)
+Add the following lines to the `Podfile` of your project and replace the `Version Number` with the desired version of the `BitmovinPlayerCore`. You can find all versions [here](https://github.com/bitmovin/cocoapod-specs/tree/master/Specs/BitmovinPlayerCore).
```

Comment:
Nit: 
```suggestion
Add the following lines to the `Podfile` of your project and replace the version number with the desired version of the `BitmovinPlayerCore`. You can find all versions [here](https://github.com/bitmovin/cocoapod-specs/tree/master/Specs/BitmovinPlayerCore).
```

Reply:
I think I would prefer the old version of it 🤔 (This is consistent with `player-ios` then)